### DISCUSSION TITLE
논의 사항(2/23)

### DISCUSSION THREAD

**Discussion by damilog** [damilog/board-sample#20] [General]
제 기록용으로 남깁니다~! 
- [ ] 이미지 받는다면 어떻게 받을 건지? -> 불가하다면 게시물 작성 시에는 이미지 넣지 않고, 기존 데이터에만 이미지 넣어서 조회하기
- [ ] 게시물 작성/수정 시 작성된 text는 어떻게 전달? body에 text 넣어서 요청보내면 되는건지?
- [ ] 맛집 카테고리 정하기 중식/양식/..
- [ ] 평점 계산 해서 내려주세요
- [ ] 네이버 플레이스 주소 넣어주세요 (수기로 해야하긴 함..ㅠㅠ)

추가 작성 예정

**Comment by MinChul-Son** [comment]
작성/수정 시 작성된 text는 `json`형식으로 `id`와 `content`등을 담아서 넘겨주시면 될 듯?
